python-package/hatch_build.py

```diff
@@ -3,16 +3,16 @@
 Here, we customize the tag of the generated wheels.
 """
 
-import sysconfig
 from typing import Any, Dict
 
 from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+from pip._vendor.packaging.tags import platform_tags
```

Comment:
Since this is just [coming from `packaging`]( https://github.com/pypa/packaging/blob/c385b58c4d0e4d66f5c8273ebb155801c92aadf3/src/packaging/tags.py#L499 ), should we use that directly

```suggestion
from packaging.tags import platform_tags
```

Reply:
No, we can't know for sure if `packaging` is available on the user's system. It's better to count down `pip`, since it's directly used for building and packaging wheels.

Reply:
I wonder if `pip` carries a significantly modified version of `packaging`. In the end, we have to generate a tag that will be accepted by `pip`. I don't want a situation where we use `packaging` to generate a tag and have it rejected by `pip`.

Reply:
I don't think so. AIUI `pip` adds several things to a `_vendor` directory to avoid having dependencies on them, which would result in cycles in the dependency graph

In fact [their vendoring policy]( https://github.com/pypa/pip/blob/ac63d0fa126fbfb01263b1597c2b425ba720d59f/src/pip/_vendor/README.rst ) says

> * Vendored libraries **MUST** not be modified except as required to
  successfully vendor them.
> * Vendored libraries **MUST** be released copies of libraries available on
  PyPI.
> ...